docs/content/en/docs/reference/packagespec/cluster-autoscaler/v9.21.0.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "v9.21.0"
+linkTitle: "v9.21.0"
+weight: 20
+description: >
+---
+
+### Configuring Cluster Autoscaler in EKS Anywhere package spec
+
+| Parameter | Description | Default |
```

Comment:
Just to confirm - all parameters here have been tested by us?

Reply:
I double checked everything and noticed that `imagePullSecrets` seems to be overwritten by the controller. I've pulled it out of the docs. I think I also have a mismatch between the schema.json and values.yaml for the value. I'll open a PR to resolve it as well.